src/About.jl

```diff
@@ -130,13 +130,15 @@ style of the output.
 module About
 
 using StyledStrings: @styled_str, Face, face!, addface!
-using JuliaSyntaxHighlighting: highlight
+using JuliaSyntaxHighlighting
 using InteractiveUtils
 
 @static if VERSION >=v"1.11-alpha"
     using Base: AnnotatedString, AnnotatedIOBuffer
+    global const highlight = identity
 else
     using StyledStrings: AnnotatedString, AnnotatedIOBuffer
+    global const highlight = JuliaSyntaxHighlighting.highlight
```

Comment:
Any reason not to write it as
```suggestion
    using JuliaSyntaxHighlighting: highlight
```
?

Reply:
just parallel code structure between the two branches -- I know they're equivalent